packages/testcontainers/src/container-runtime/clients/container/docker-container-client.ts

```diff
@@ -158,6 +158,7 @@ export class DockerContainerClient implements ContainerClient {
     try {
       log.debug(`Fetching container logs...`, { containerId: container.id });
       const stream = (await container.logs({
+        ...opts,
```

Comment:
added ability to pass all params to logs

earlier we can pass only since option

Reply:
What will happen if the user of this method passes opts with follow, stdout and/or stderr with false value? Will those values be overridden by the defaults in https://github.com/testcontainers/testcontainers-node/pull/670/files#diff-cde64c872b844fe1a7e8949d7eee46cea0cb346b40d840597c1a4785d68998d2R162-R164?

If so, it will cause an unexpected behaviour on the user side, as the values won't be applied but overridden by the library.

I'm not a Typescript developer, so sorry if this is an incorrect observation from my side.

If that's the case I'd go with defining a fallback/default opts variable that will take the current default values if the opts passed by the users are empty/invalid/null. Wdyt?

Reply:
reverted to previous version